R/print.data.table.R

```diff
@@ -93,7 +94,7 @@ print.data.table = function(x, topn=getOption("datatable.print.topn"),
   if (quote) colnames(toprint) <- paste0('"', old <- colnames(toprint), '"')
   if (isTRUE(trunc.cols)) {
     # allow truncation of columns to print only what will fit in console PR #4074
-    widths = dt_width(toprint, class, row.names, col.names)
+    widths = dt_width(toprint, n_x, class, row.names, col.names)
     cons_width = getOption("width")
     cols_to_print = widths < cons_width
```

Comment:
I really feel this line should be `widths <= cons_width` but `print.default` appears to wrap at equality... an issue with `print.default`?

Reply:
Yeah, I saw that too but thought maybe that was expected behavior. It’s good to know that is the behavior we are working with.